Source/Core/Common/FileUtil.h

```diff
@@ -136,12 +136,18 @@ bool CreateDir(const std::string& filename);
 // Creates the full path of fullPath returns true on success
 bool CreateFullPath(const std::string& fullPath);
 
+enum class IfAbsent
```

Comment:
IfAbsentBehavior perhaps? That's a bit more verbose but I don't think IfAbsent is clear enough; it doesn't sound like the name of an enum class.

Reply:
Done.